### PR TITLE
Remove annoying gray line

### PR DESCRIPTION
**Pull request by grosser**
same as https://github.com/zendesk/samson/pull/3921

but:
- global
- symmetric since we don't mess with the baseline

Screen Shot 2021-01-28 at 9 01 49 AM
Screen Shot 2021-01-28 at 9 01 47 AM

@zendesk/compute

### CODE REVIEW

**Review by eatwithforks — APPROVED** [review]

**Review by tsunny — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -685,7 +685,7 @@ DEPENDENCIES
   pry-rails
   pry-rescue
   pry-stack_explorer
-  puma (= 4.3.6)
+  puma
```
Comment (uthark) [inline Gemfile.lock:688]: This change is not related to the PR. :)

**Review by uthark — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (grosser) [reply Gemfile.lock:688]: yeah, someone changed gemfile and did not bundle
